runelite-client/src/main/java/net/runelite/client/plugins/attackstyles/AttackStylesPlugin.java

```diff
@@ -177,6 +188,29 @@ public void onGameStateChange(GameStateChanged event)
 			updateWarnedSkills(config.warnForRanged(), Skill.RANGED);
 			updateWarnedSkills(config.warnForMagic(), Skill.MAGIC);
 		}
+
+		if (event.getGameState() == GameState.LOGIN_SCREEN ||
+			event.getGameState() == GameState.HOPPING ||
+			event.getGameState() == GameState.CONNECTION_LOST)
+		{
+			loginMessageSent = false;
+			messageTickGuarded = true;
+		}
+	}
+
+	@Subscribe
+	public void onGameTick(GameTick e)
+	{
+		// Utilizes GameTicks, seeing as variables are set PRIOR to a GameTick actually being ran.
```

Comment:
I'm inclined to say this should be part of the commit body rather than a comment. It should definitely be stated somewhere, but the full explanation probably doesn't need to be here. A short `// One tick delay to ensure vars are set`` comment here should suffice if this is moved to the commit message.

Reply:
Okay. I figured someone wouldn't want it there, but, it was mainly for someone to just understand why it utilizes that method. I'll move it.